applications/openshift/api-server/api_server_tls_security_profile_custom_min_tls_version/kubernetes/shared.yml

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_ocp
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+    name: cluster
+spec:
+    tlsSecurityProfile:
+        Custom:
+            minTLSVersion: VersionTLS12
```

Comment:
I'm on the fence whether we should be adding any remediation at all, but if we do, I think it should be sticking with the defaults, definitely not adding any custom config.
What do you think @Vincent056 ?

Reply:
Let's keep it for now, but my opinion is that we should just roll with the defaults.